slc/component/matter-core-sdk/tracing.slcc

```diff
@@ -38,6 +38,13 @@ define:
   - name: MATTER_TRACING_ENABLED
     value: 1
   - name: MATTER_TRACING_BUILD_CONFIG_H_
+  - name: TRACING_RUNTIME_STATS
```

Comment:
Side comment about TRACING_RUNTIME_STATS, we should have a matter_tracing_statistics component that can only hold that define. This would enable activating the tracing statistics through a build argument.